### DISCUSSION TITLE
Is there a general problem with the ics integration of Abfall.IO / AbfallPlus.de?

### DISCUSSION THREAD

**Discussion by SamuSnorre** [mampfes/hacs_waste_collection_schedule#2849] [Q&A] [answered]
I used the hardcore version to determine the data from the provider in the documentation and entered it. 

![website](https://github.com/user-attachments/assets/86a102a0-3243-40e4-b8cc-a0bfb629204d)
![eingabe](https://github.com/user-attachments/assets/8f2a5b99-95dc-47cd-aabd-3b8a57501608)

The result is a large error message.

![fehler](https://github.com/user-attachments/assets/53ac163b-328f-460c-8f39-e3d4afe1bad7)

Is there any idea where the problem could lie?

Many thanks and best regards
Stefan

**Comment by 5ila5** [comment, marked as answer]
They use the new API, so the abfall_io source does not work. You can use the ICS source there is [documentation for abfall.io with the new overlay](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/ics/abfall_io_ics.md) (there is even a link in the README for Abfallwirtschaft Landkreis Böblingen)

Or you can use the [backend source for the app](https://github.com/mampfes/hacs_waste_collection_schedule/blob/2.3.0/doc/source/app_abfallplus_de.md)

**Comment by SamuSnorre** [comment]
Many thanks for the tip. It now works with ics.
